src/controllers/settings.js

```diff
@@ -0,0 +1,20 @@
+const getUser = require('../database/queries/getData');
+
+exports.get = (request, response) => {
+  if (request.auth) {
+    const userId = request.auth.id;
+    getUser.getUser(userId)
+      .then((result) => {
+        response.render('settings', {
```

Comment:
@Angham116 I wonder if you should also handle the case of the getUser() call returning 0 rows by responding with an error.

Reply:
@tomduggan85  do you mean 
```
if (result.rows[0]) {
          response.render('settings', {
            js: ['helpers/collectData', 'settings'],
            userInfo: result.rows[0],
            css: ['settings'],
            layout: 'main',
            title: 'settings',
          });
        }
```  
??

Reply:
@Angham116 Yep! And also an else:
```
if (result.rows[0]) {
  response.render('settings', {
  ...
  ...
} else {
  response.status(400).send({ Error: 'No user found' });
}
```